client/task-list/tasks.js

```diff
@@ -58,95 +58,79 @@ export function getAllTasks( {
 	);
 
 	const tasks = [
+		{
+			key: 'store_details',
+			title: __( 'Store details', 'woocommerce-admin' ),
+			container: null,
+			onClick: () => {
+				window.location = getAdminLink(
+					'admin.php?page=wc-admin&reset_profiler=1'
+				);
+			},
+			completed: profileItems.completed,
+			visible: true,
+			time: __( '4 minutes', 'woocommerce-admin' ),
+		},
 		{
 			key: 'purchase',
 			title: __( 'Purchase & install extensions', 'woocommerce-admin' ),
-			content: __(
-				'Purchase, install, and manage your extensions directly from your dashboard',
-				'wooocommerce-admin'
-			),
-			icon: 'extension',
 			container: null,
 			onClick: () =>
 				remainingProductIds.length ? toggleCartModal() : null,
 			visible: productIds.length,
```

Comment:
Clicking "Purchase & install extensions" doesn't do anything for me. Maybe this should be `visible: remainingProductIds.length`?

Reply:
I think it's functioning correctly (this is unchanged in the PR) - is it showing complete? If you selected a plugin and have it installed, the step will display as completed.

Reply:
Ok, maybe it because I'm doing odd things like `const isTaskListEnabled = true;` hard coded to get it to come up.